### DISCUSSION TITLE
Add support for groups

### DISCUSSION THREAD

**Discussion by SkypLabs** [ente-io/ente#482] [Enhancements]
Hi,

I'd like to suggest adding groups to easily display only personal or work items for example. You can have a look at how Aegis does it for inspiration.

And thanks for this great tool!

Cheers

**Comment by itInner** [comment]
I agree that grouping is too easy to use for a large number of scenarios, and it is highly recommended that developers consider this feature
Also thanks guys,  for this great tool!

**Comment by 0uros** [comment]
I would love for this to be added soon! It's something that has 2FAS Auth, and I love it

**Comment by Bayleyvilla** [comment]
Please Please Please : add groups as this will be such an assist when having large numbers of keys to manage.
The option for the end user to name the groups.
Have a look at https://github.com/jamie-mh/AuthenticatorPro this is for android no cross platform but grouping is good.
Hope this is not going to fall on deaf ears.

**Comment by SkypLabs** [comment]
Version 3.0 introduced tags. So, I will close this discussion as resolved.
